### PR TITLE
New market creation end time display in review

### PR DESCRIPTION
**Pull request by bthaile**
design agreed to adding both timezone date time and UTC date time to review page.

Added new date time display, showing both timezone time and utc time, fixed a few types, moved complexity of building endTime formatted to format-date utility

https://github.com/AugurProject/augur/issues/3109

fixes user selecting date time logic, a little clean up.

image

### CODE REVIEW

File: packages/augur-ui/src/modules/create-market/components/form-details.tsx [hunk 1]
```diff
@@ -127,42 +125,41 @@ export default class FormDetails extends React.Component<
...
               minute={minute}
               meridiem={meridiem}
-              onChange={(label: string, value: string) => {
+              onChange={(label: string, value: number) => {
                 onChange(label, value)
               }}
               onFocusChange= {(focused: Boolean) => {
                 if (!hour) {
-                  onChange("hour", "12");
+                  onChange("hour", 12);
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/form-details.tsx]: time selector won't let you type "0"

Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/form-details.tsx]: clicking on the time selector doesn't fill in the values as supposed to

Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/form-details.tsx]: i'm also not seeing the validations for time show up

File: packages/augur-ui/src/modules/create-market/components/form-details.tsx [hunk 2]
```diff
@@ -172,12 +169,12 @@ export default class FormDetails extends React.Component<
               focused={s.timeFocused}
               errorMessage={validations[currentStep].hour}
             />
-            <TimezoneDropdown onChange={(value: number, offset: string) => {
-              onChange("offsetName", value);
+            <TimezoneDropdown onChange={(offsetName: string, offset: number, timezone: string) => {
               onChange("offset", offset)
-            }} timestamp={endTime} timezone={offsetName} />
+              onChange("timezone", timezone)
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/form-details.tsx:174]: i don't see timezone being used at all?

**Comment by bthaile** [comment]
I’ll take a look and design want 4px between the times

File: packages/augur-ui/src/modules/create-market/components/common.styles.less [hunk 3]
```diff
@@ -212,7 +212,10 @@
 		.text-16-medium;
 
 		color: @color-alternate-info-text;
-	}
+  }
+  > span:last-of-type {
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/create-market/components/common.styles.less]: this isn't right. it will effect all the other small subheaders

**Review by phoebemirman — CHANGES_REQUESTED** [review]
see comments, also not sure if it is showing the times correct still

File: packages/augur-ui/src/modules/create-market/components/form-details.tsx (hunk 1 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/create-market/components/form-details.tsx]: also "0" isn't valid for minutes in the time selector. which makes it a bit messed up

File: packages/augur-ui/src/modules/create-market/components/form-details.tsx (hunk 2 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/form-details.tsx:174]: it is on my branch and everything is checked in. I'll see

Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/form-details.tsx:174]: everything is checked in
image

File: packages/augur-ui/src/modules/create-market/components/form-details.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/form-details.tsx]: hour is defaulted to 12, I'll change that back to null

File: packages/augur-ui/src/modules/create-market/components/form-details.tsx (hunk 2 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/create-market/components/form-details.tsx:174]: but then timezone is never used anywhere? don't see why it needs to get set

File: packages/augur-ui/src/modules/create-market/components/common.styles.less (hunk 3 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/common.styles.less]: what is the right way?

Comment (phoebemirman) [reply packages/augur-ui/src/modules/create-market/components/common.styles.less]: do it in the individual time component or > span:nth-of-type(3) could work.

File: packages/augur-ui/src/modules/create-market/components/form-details.tsx (hunk 2 above)
Comment (phoebemirman) [reply packages/augur-ui/src/modules/create-market/components/form-details.tsx:174]: you don't need both offsetName and timezone

**Review by phoebemirman — CHANGES_REQUESTED** [review]
not seeing time validations:
Screen Shot 2019-08-08 at 10 09 12 AM

needs to set time when clicked on selector, and shouldn't be showing a single zero here at this time:
Screen Shot 2019-08-08 at 10 09 02 AM

File: packages/augur-ui/src/modules/create-market/components/form-details.tsx (hunk 2 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/form-details.tsx:174]: timezone is needed only to pick the correct item in the timezone dropdown. True, only the parts of the selected timezone is used.

**Review by phoebemirman — APPROVED** [review]
need to fix the timezone for real in other PR

**Pull request merged**